build_debian.sh

```diff
@@ -132,6 +125,14 @@ sudo chroot $FILESYSTEM_ROOT update-initramfs -u
 ## Install latest intel igb driver
 sudo cp target/debs/igb.ko $FILESYSTEM_ROOT/lib/modules/3.16.0-4-amd64/kernel/drivers/net/ethernet/intel/igb/igb.ko
 
+## Install package without starting service
+trap_push 'sudo rm -f $FILESYSTEM_ROOT/usr/sbin/policy-rc.d'
+sudo tee -a $FILESYSTEM_ROOT/usr/sbin/policy-rc.d > /dev/null <<EOF
+#!/bin/sh
+exit 1
+EOF
+sudo chmod a+x $FILESYSTEM_ROOT/usr/sbin/policy-rc.d
```

Comment:
Actually trap_push line will remove this file, either on normal exit or on error.

---
In reply to: [94189805](https://github.com/Azure/sonic-buildimage/pull/171#discussion_r94189805) [](ancestors = 94189805)